python/requirements.txt

```diff
@@ -1,2 +1,2 @@
 # This file should list any python package dependencies.
-scikit-learn==0.18.1
+scikit-learn>=0.18.1
```

Comment:
Do we also want to impose a version upper bound?

Reply:
That's a good idea, I'll set an upper bound of 0.19 (the latest release) for now.